Short-Answer/Algorithms_Answers.md

```diff
@@ -1,15 +1,41 @@
-#### Please add your answers to the ***Analysis of  Algorithms*** exercises here.
+#### Please add your answers to the **_Analysis of Algorithms_** exercises here.
+
+<!-- Give an analysis of the running time of each snippet of
+pseudocode with respect to the input size n of each of the following: -->
 
 ## Exercise I
```

Comment:
Great job on scoring 2 out of 3 possible points on Exercise I, @MelvinPeepers! 👍  
<img width="1280" alt="melvin_short_answer" src="https://user-images.githubusercontent.com/16110931/76693666-5b5cc280-663f-11ea-8fe4-197ff3dcc2b5.png">

Reply:
<img width="980" alt="Screen Shot 2020-03-14 at 10 09 56 PM" src="https://user-images.githubusercontent.com/16110931/76693739-a3c8b000-6640-11ea-9c5d-82972d2dfc8e.png">